add thank you reviewers blog (#348)
pythonindia/inpycon-blog: content/blog/thank-you-reviewers-2020.md

Title: Thank you PyCon India 2020 reviewers 
Date: 2020-12-13 
Category: 2020 
Slug: thank-you-reviewers-2020 
Summary: Thanking our humble reviewers who reviewed talks and workshop proposals submitted to PyCon India 2020.

> 
> 
> 

Reviewer: ```suggestion

```

[Linkedin](https://linkedin.com/in/stalwartcoder) 
[Twitter](https://twitter.com/stalwartcoder)

### Akash Mahajan
    Akash help CTOs & SREs with Kubernetes and Container security compliances | CKA, OSCP | Author - Security Automation using Ansible, Burp Suite Essentials
[Twitter](https://twitter.com/makash)

### Anant shrivastava
    Anant Shrivastava is an information security professional with 12+ yrs of corporate experience with expertise in Network, Mobile, Application and Linux Security. Anant is an avid opensource supporter and runs multiple open source projects prominent of them being Android Tamer and CodeVigilant.
[Twitter](https://twitter.com/anantshri)

### Anirudha K
     Anirudha has been active in python community with co-organising BangPypers and DjangoGirls Bangalore. Also cofounded the largest blockchain community in India, Blockchained India. Apart from building blockchain and fintech products as Acyclic Labs, Anirudha also does consulting on backend and product architecture. 
[website](https://anirudha.dev/) 

> [Twitter](https://twitter.com/kranirudha).

Reviewer: ```suggestion
[Twitter](https://twitter.com/kranirudha)
```

### Charmi Chokshi
    Charmi is a Machine Learning Engineer and one of the youngest Google Developers Expert in ML. She loves helping developers & students looking forward to growing their careers in the field of Artificial Intelligence via tech talks, mentoring sessions, and blogs.
[Twitter](https://twitter.com/CharmiChokshi)

### Chetan Giridhar
    Chetan is a Python enthusiast and open source evangelist. He has been a speaker at Python conferences and takes keen interest in platform engineering, distributed systems and real time cloud communications. In his spare time Chetan reads about the world history and loves helping startups.

### Derrick Joseph
     Derrick is former PyCon speaker and has been a Pythonista since 2015, he started his career as a Data Scientist and later moved to Engineering. He has experience working with ML, Systems and has implemented business processes automations as generic frameworks.

> His current area of exploration is CPython and is working with Linkedin's Production Infrastructure Engineering team.

Reviewer: Line break not needed.

>      Derrick is former PyCon speaker and has been a Pythonista since 2015, he started his career as a Data Scientist and later moved to Engineering. He has experience working with ML, Systems and has implemented business processes automations as generic frameworks.
> His current area of exploration is CPython and is working with Linkedin's Production Infrastructure Engineering team.
> [Twitter](https://twitter.com/C0d3eswaran)

Reviewer: ```suggestion
     Derrick is former PyCon speaker and has been a Pythonista since 2015, he started his career as a Data Scientist and later moved to Engineering. He has experience working with ML, Systems and has implemented business processes automations as generic frameworks. His current area of exploration is CPython and is working with Linkedin's Production Infrastructure Engineering team.
[Twitter](https://twitter.com/C0d3eswaran)
```

> Title: Thank you PyCon India 2020 reviewers Date: 2020-12-13 Category: 2020 Slug: thank-you-reviewers-2020 Summary: Thank you PyCon India 2020 reviewers

Reviewer: ```suggestion
Title: Thank you PyCon India 2020 reviewers 
Date: 2020-12-13 
Category: 2020 
Slug: thank-you-reviewers-2020 
Summary: Thanking our humble reviewers who reviewed talks and workshop proposals submitted to PyCon India 2020.
```

> We would like to express our appreciation and gratitude to the following people who participated in the review panel at PyCon India 2020. The talk selection process would not have been possible without their effort and expertise. The review panel comprised of a few more people, who have chosen to stay anonymous. Many thanks to them too.

Reviewer: ```suggestion
We would like to express our appreciation and gratitude for the following people who participated in the review panel at PyCon India 2020. The talk selection process would not have been possible without their effort, expertise, and time they committed to reviewing each talk. The review panel comprised of a few people, who have chosen to stay anonymous. Many thanks to them too.
```

[Twitter](https://twitter.com/abhicantdraw) 

### Abhishek Dash
    Abhisek is a doctoral student in CSE department at IIT Kharagpur. He is currently a member of Complex Network Research Group (CNeRG) at IIT Kharagpur. His research interests lie in the domain of fairness in machine learning and platform governance. 
[Twitter](https://twitter.com/AbhisekFair)

### Abhishek Mishra
     He is a backend engineer currently working at Saama Technologies. He mostly works on python and evangelizes for the same.

> He is a organizers of GDG Chennai and a volunteer in Chennaipy, Pyladies chennai meetup group. He has been in PyCon India team since 2019.

Reviewer: Remove line, make it one para for it took better?

[Linkedin](https://linkedin.com/in/stalwartcoder) 
[Twitter](https://twitter.com/stalwartcoder)

### Akash Mahajan
    Akash help CTOs & SREs with Kubernetes and Container security compliances | CKA, OSCP | Author - Security Automation using Ansible, Burp Suite Essentials
[Twitter](https://twitter.com/makash)

### Anant shrivastava
    Anant Shrivastava is an information security professional with 12+ yrs of corporate experience with expertise in Network, Mobile, Application and Linux Security. Anant is an avid opensource supporter and runs multiple open source projects prominent of them being Android Tamer and CodeVigilant.
[Twitter](https://twitter.com/anantshri)

> ### Anirudh K

Reviewer: Anirudh or Anirudha?

### Derrick Joseph
     Derrick is former PyCon speaker and has been a Pythonista since 2015, he started his career as a Data Scientist and later moved to Engineering. He has experience working with ML, Systems and has implemented business processes automations as generic frameworks.
His current area of exploration is CPython and is working with Linkedin's Production Infrastructure Engineering team.
[Twitter](https://twitter.com/C0d3eswaran)
[Linkedin]( https://www.linkedin.com/in/derrickjosephknatt)

### Dhruva Shashtri 
    Her interests as an inquisitive #WomenInTech is to learn new every day so she continually looks out for areas that interest her to work on. Being a facilitator for #IamRemarkable initiative and community passionate, playing a role as an Ambassador at WTM Ahmedabad, enthusiastic about emboldening women in every sphere of their life and enjoys people’s synergy.
[Twitter](https://twitter.com/dhruvashastri12)

### Girish Joshi

>     Girish  is a polyglot programmer and a free and open source enthusiast from Pune. He has been a part of the Python Pune community since 2015.

Reviewer: ```suggestion
    Girish is a polyglot programmer and a free and open source enthusiast from Pune. He has been a part of the Python Pune community since 2015.
```

> For many years, he has been hacking on embedded systems and IoT applications.

Reviewer: Also make it one para maybe?

### Kuldeep Rishi
    I am a husband, traveller and a software engineer working on building a hyper-scale social engagement platform at Disney+Hotstar for millions of users around the world. I like to automate repetitive things and solve complex problems with simple solutions to make life easier for others. 
[Twitter](https://twitter.com/kuldeeprishi)

### Manivannan E
    Manivannan is an open source evangelist & a python programmer; He has 17+ years of experience in software development and has performed various roles in organizations. Outside his regular job, he is also a python community leader & organizer in Hyderabad & helped revive the community. He chaired multiple python conferences in India - PyConf Hyd 2017, Pycon India 2018 & Pyconf Hyd 2019. Currently, he is working as a Director in Lifion by ADP Inc based out of NY, building next gen products.
[Twitter](https://twitter.com/ManithaE)

> 

Reviewer: ```suggestion
```

 [Twitter](https://twitter.com/runcyoommen)

### Sahil Dua
    Sahil Dua is a software engineer at Google. He has been an active speaker at tech conferences for the last 4 years on various topics like data analysis using pandas, A/B testing for product development and scaling machine learning predictions infrastructure using Kubernetes. Sahil is a passionate open source contributor and has contributed to some famous projects such as git, pandas, hound, Go-Github, and so on. He's also been an open source community leader for over 2 years at DuckDuckGo.
[Twitter](https://twitter.com/sahildua2305)

### Shivani Bhardwaj 
    Shivani is one of the rare developers who do not need caffeine to function. She currently works with The Open Information Security Foundation developing and learning about Suricata - A Network Security

> Monitoring tool. She has been a member of several Open Source communities around the world and is a proud supporter of women in STEM. She got associated with Python and its communities across the country in 2017; and remains a member to the date. She has no particular place of stay but can be found in the mountains or at the beaches for most of the year.

Reviewer: Need to remove line break

[Twitter](https://twitter.com/msukanya93)

### Supreet Sethi
    Supreet tenured technical professional with significant experience in Python. He has worked for companies like Yahoo!, Jabong and Tokopedia. Currently, he works for Amazon AWS as Principal Solution Architect focusing on bringing technology best practices to Startups in ASEAN region.
[Twitter](http://twitter.com/djinn)

### Usha Rengaraju 
    Usha is India’s first women Kaggle Grandmaster and she is ranked as top ten Data Scientists in India for the year 2020 by Analytics India Magazine.She is Data Science Global Ambassador for HP and NVIDIA.
She organised NeuroAI (www.neuroai.in) which is India’s first-ever research symposium in the interface of Neuroscience and Data Science. She specialises in Probabilistic Graphical Models, Machine Learning and Deep Learning. She has prepared curriculum for BITS Pilani’s masters in Data Science program (consumed by 20,000+ students ) and Upgrad’s PGP program in DS(consumed by 10,000+ students).
[Twitter](https://twitter.com/URengaraju)

>  ### Vinita Sharma

Reviewer: ```suggestion
### Vinita Sharma
```

    I work as a member of technical staff at Kubric and have over 6+ years of experience working across startups including Myntra and Jumbotail.

> When I am not building systems or learning about the latest stuff, I will mostly be painting or playing chess. 

Reviewer: One para should look good here.
